src/libsyntax/print/pp.rs

```diff
@@ -507,8 +507,14 @@ impl<'a> Printer<'a> {
         }
     }
     pub fn print_newline(&mut self, amount: isize) -> io::Result<()> {
+        #[cfg(windows)]
+        const NEWLINE: &'static str = "\r\n";
+
+        #[cfg(not(windows))]
+        const NEWLINE: &'static str = "\n";
```

Comment:
Yeah that'd do the trick! You could also use `b"\n"` to avoid the call to `.as_bytes()`